deno2/BUILD.gn

```diff
@@ -85,20 +106,12 @@ proto_library("msg_proto") {
   generate_python = false
 }
 
-template("run_node") {
-  action(target_name) {
-    forward_variables_from(invoker, "*")
-    script = "js/run_node.py"
-  }
-}
-
 run_node("bundle") {
   out_dir = "$target_gen_dir/bundle/"
   sources = [
     "js/main.ts",
     "js/msg.pb.d.ts",
     "js/msg.pb.js",
-    "js/package.json", # `browserslist` field controls transform.
```

Comment:
@ry why remove this? I don't agree, I just put that there because ninja wasn't rebuilding the bundle when it should.

Reply:
Oh - that wasn't intentional.

Reply:
Ok, I put it back.